packages/client-sdk-nodejs/test/integration/integration-setup.ts

```diff
@@ -249,11 +254,20 @@ export function SetupStorageIntegrationTest(): {
   storageClient: PreviewStorageClient;
   integrationTestStoreName: string;
 } {
-  const {integrationTestCacheName} = SetupIntegrationTest();
+  const integrationTestStoreName = testStoreName();
   const storageClient = momentoStorageClientForTesting();
+
+  beforeAll(async () => {
+    await createStoreIfNotExists(storageClient, integrationTestStoreName);
+  });
+
+  afterAll(async () => {
+    await deleteStoreIfExists(storageClient, integrationTestStoreName);
+  });
```

Comment:
added a `beforeAll` and `afterAll` to create a shared store that can be used by storage data plane tests. this will prevent leaked stores and ensure minimal creation of stores where possible.